modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js

```diff
@@ -292,11 +307,24 @@ function ActionsDropdownRenderer({actions, itemData, itemId}) {
 			action.label
 		);
 
+		const onActionDropdownItemClick = context.onActionDropdownItemClick;
+
```

Comment:
Not a block, but I guess you could do 

```js
const {onActionDropdownItemClick} = context;
```

(As far as I'm concerned, I'm not a fan of destructuring, but just wanted to comment in case you'd prefer it)

Reply:
IMO, deconstruct is nice to use to avoid the `context` variable, with something like 
```js
const {onActionDropdownItemClick, ...} = useContext(DataSetContext);
```
In this case `context` was already available, since we are passing it to `handleAction`. All of this could probably be simplified, but I tried to avoid unnecessary SF in this PR. The amount of changes is already quite high.

Reply:
Make sense, thanks for the explanation.